### ISSUE TITLE
Setting 'autoOk' should hide the Picker's buttons

### ISSUE THREAD

**Issue by kr3zw** [dmtrKovalenko/material-ui-pickers#372]
## Environment

| Tech                | Version |
|---------------------|---------|
| material-ui-pickers | 1.0.0-rc.6 |
| material-ui         | 1.0.0-beta.42 |
| React               | 16.2.0 |
| Browser             | Any |
| Platform            | Windows |

## Steps to reproduce
1. Set autoOk to true

## Expected behavior
![expected](https://user-images.githubusercontent.com/19870633/39124487-510fa7cc-4704-11e8-96aa-e3620bc6c7d8.jpg)

## Actual behavior
![full](https://user-images.githubusercontent.com/19870633/39124490-55906f02-4704-11e8-9d0c-3751ede69fcb.jpg)

Since autoOk is on, you can't select a date and click OK.
Therefore both buttons should be hidden when setting the prop into true - better UI/UX

**Comment by cherniavskii** [comment]
But you should be able to click Cancel.
Or Clear, if `clearable` prop is set to true ;)

**Comment by kr3zw** [comment]
I agree that there's a point on showing the buttons when having the `Clear` or `Today` props set to true.

Though, when the only buttons you got are `Cancel` and `OK`, I believe you could just not show them. Clicking outside the picker closes it (_replacing the Cancel button_) and having the **autoOk** prop set to true (_replacing the OK button_), make them not needed.
